disnake/ext/commands/context.py

```diff
@@ -32,6 +32,8 @@
 import disnake.utils
 from disnake.message import Message
 
+from ... import ApplicationCommandInteraction
```

Comment:
probably should use `from disnake import ApplicationCommandInteraction`

Reply:
https://github.com/DisnakeDev/disnake/pull/398/commits/410a7d049c893ccc7ed30a00409661e6fd7c5d13